frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -863,15 +859,54 @@ const HostDetailsPage = ({
     name: host?.mdm.macos_setup?.bootstrap_package_name,
   };
 
+  // host.platform = "windows";
+
   const isDarwinHost = host.platform === "darwin";
   const isIosOrIpadosHost = isIPadOrIPhone(host.platform);
   const isAndroidHost = isAndroid(host.platform);
 
-  const detailsPanelClass = classNames(`${baseClass}__details-panel`, {
-    [`${baseClass}__details-panel--ios-grid`]: isIosOrIpadosHost,
-    [`${baseClass}__details-panel--android-grid`]: isAndroidHost,
-    [`${baseClass}__details-panel--macos-grid`]: isDarwinHost,
-  });
+  const testEndUserData: IHostEndUser[] = [
```

Comment:
What do you think about leaving the existing device mapping API stuff in for the time being and adding an temporary helper that transforms the device mapping response to the shape of the new end users response? My concern is just that we'll lose the current device mapping UI on main while we're waiting for the rest of the feature. 

If that seems like too much effort, let's check with George and make sure they don't deploy main to dogfood until the necessary parts of the new API are working.

Reply:
Lets chat with George first. I'd prefer to leave it out as there were a lot of different places i had to remove device mapping code.